### DISCUSSION TITLE
Demo app on 22.2 with unified task list

### DISCUSSION THREAD

**Discussion by ahmedattiq** [flowsforapex/apex-flowsforapex#586] [Q&A]
Have you release a demo app on Oracle Apex 22.2 having unified task list?

**Comment by rallen2010** [comment]
No, we haven't so far.  Are there specific things from the APEX Office Hours demo that you want to see - for example, how did we merge the Flows for APEX Inbox and the APEX Universal Task Inbox?

**Reply by ahmedattiq** [reply]
Hi @rallen2010 i am able to merge now and can see approval items in the unified task list. 

I wanted to see a demo app to understand in more detail which can easily be replicated to my own apex app.
